edu-services/cm-service/cm-impl/hibernate-impl/impl/src/java/org/sakaiproject/coursemanagement/impl/CourseManagementServiceFederatedImpl.java

```diff
@@ -729,13 +748,35 @@ public boolean isEnrolled(String userId, String enrollmentSetEid) {
 		return sectionRoleMap;
 	}
 
+	public Map<String, String> findSectionRoles(String userEid, String academicSessionEid) {
+		Map<String, String> sectionRoleMap = new HashMap<>();
+		for (CourseManagementService cm : implList) {
+			Map map = cm.findSectionRoles(userEid, academicSessionEid);
+			if (map == null) {
+				continue;
+			}
+			for (Iterator mapIter = map.keySet().iterator(); mapIter.hasNext();) {
```

Comment:
this should use a newer loop construct, Iterator is never used.

Reply:
Yeah I changed all the others in this patch, must have just overlooked this one. Thanks.

Reply:
@ern, fixed that up. Thanks for catching it!